aiohttp/log.py

```diff
@@ -1,7 +1,6 @@
 import logging
 
 
-access_logger = logging.getLogger('aiohttp.access')
```

Comment:
I saw a project that used this variable for logging.

Is whole `aiohttp.log` should be considered as  private module?